src/radial.js

```diff
@@ -180,9 +212,15 @@
 				j++;
 				count++;
 			}
-			for(j; j < count; j++) {
-				var newalfa = Math.round(i + alfa);
-				this._items[j]._alfa = newalfa;
+			for(j; j < this._items.length; j++) {
+				if(newcount != 0){
+					var newalfa = (this._items[j]._alfa == 0) ? Math.round(i + alfa) : Math.round(i + this._items[j]._alfa);
+					this._items[j]._alfa = newalfa;
+				}
+				else{
```

Comment:
Don't like this kind of identation but don't worry :)

`} else {`
better than
```
}
 else {
```